shared/wallets/send-form/note-and-memo/container.tsx

```diff
@@ -10,9 +10,12 @@ export const SecretNote = namedConnect(
     const recipientType = state.wallets.building.recipientType
     const building = state.wallets.building
     const built = building.isRequest ? state.wallets.builtRequest : state.wallets.builtPayment
-    const maxLength = building.isRequest
-      ? state.wallets.staticConfig.requestNoteMaxLength
-      : state.wallets.staticConfig.paymentNoteMaxLength
+    // TODO is this ok
+    const maxLength = state.wallets.staticConfig
```

Comment:
config can be null

Reply:
I think this is what we want to do. `staticConfig` should be set by an RPC at load, but in the event it isn't, I think 0 is a fine value to default to.